cpp/src/umap/knn_graph/algo.cuh

```diff
@@ -26,10 +26,15 @@
 #include <selection/knn.cuh>
 
 #include <raft/cudart_utils.h>
+#include <raft/mr/device/allocator.hpp>
 
 #include <raft/sparse/cusparse_wrappers.h>
 #include <raft/error.hpp>
 
+namespace ML {
+using deviceAllocator = raft::mr::device::allocator;
+}
```

Comment:
This seems like another way the PR is dealing with the same issue, I think consistency in the PR would be good unless it would be way too much work

Reply:
Here, I think I was explicitly trying to minimize the diff, but if we're cool with swapping it over to always reference `raft::mr::device::allocator`, I'd be in favor.